packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/index.js

```diff
@@ -233,9 +233,20 @@ const EditViewDataManagerProvider = ({
     ({ target: { name, value, type } }, shouldSetInitialValue = false) => {
       let inputValue = value;
 
-      // Empty string is not a valid date,
-      // Set the date to null when it's empty
-      if (type === 'date' && value === '') {
+      // Allow to reset text, string, email, uid, select/enum, date
+      if (
+        (type === 'text' ||
+          type === 'string' ||
+          type === 'email' ||
+          type === 'uid' ||
+          type === 'select' ||
+          type === 'select-one') &&
+        value === ''
+      ) {
+        inputValue = null;
+      }
+
+      if (type === 'number' && !value) {
```

Comment:
I think this could be part of the previous check, if you would change `&& value === ''` to `&& !value` since `!!'' === false`.

Reply:
will do!
discovered a bug with the number input though 
if you click the input, doesn't type anything in it, then go using the arrow to change the value this happens:

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/71838159/175332132-e0526a62-4205-4e61-895b-177e7676f0cc.png">

I'll write a card in the DS backlog!